intelmq/tests/bots/outputs/elasticsearch/test_output.py

```diff
@@ -53,6 +135,131 @@ def test_event(self):
         self.con.delete(index='intelmq', doc_type='events', id=result['_id'])
         self.assertDictEqual(OUTPUT1, result['_source'])
 
+    def test_raise_when_no_template(self):
+        """
+        Test that a bot raises a RuntimeError if 'rotate_index' is set, but a matching template doesn't exist in ES.
+        """
+        self.sysconfig = {"flatten_fields": "extra",
+                          "elastic_index": "intelmq",
+                          "elastic_doctype": "events",
+                          "rotate_index": "daily"}
+        self.assertRaises(RuntimeError, self.run_bot())
```

Comment:
I now noticed that this probably does not work as intended: `self.run_bot()` is run before `assertRaises` and thus the call is `self.assertRaises(RuntimeError, None)`. Because of a bug in `assertRaises` in python < 3.8, this does not error, but breaks with Python 3.8:

```
FAIL: Test that a bot raises a RuntimeError if 'rotate_index' is set, but a matching template doesn't exist in ES.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/wagner-certat/intelmq/intelmq/tests/bots/outputs/elasticsearch/test_output.py", line 146, in test_raise_when_no_template
    self.assertRaises(RuntimeError, self.run_bot)
AssertionError: RuntimeError not raised by run_bot
```
https://travis-ci.org/wagner-certat/intelmq/jobs/622711676#L808

@stone-z is it possible for you to look into this?